cmake/profiles/common.cmake

```diff
@@ -0,0 +1,136 @@
+if(MBED_TOOLCHAIN strequal "GCC_ARM")
```

Comment:
Please add license to cmake files (just 2 lines are sufficient as in other cmake files)